packages/dcos-integration-test/extra/integration_test.py

```diff
@@ -1047,18 +1048,18 @@ def test_if_minuteman_routes_to_vip(cluster, timeout=125):
         'id': "/integration-test-app-with-minuteman-vip-%s" % test_uuid,
         'cpus': 0.1,
         'mem': 128,
-        'ports': [10000],
         'cmd': 'touch imok && /opt/mesosphere/bin/python -mhttp.server ${PORT0}',
         'portDefinitions': [
             {
-                'port': 10000,
+                'port': 0,
                 'protocol': 'tcp',
                 'name': 'test',
                 'labels': {
                     'VIP_0': '1.2.3.4:5000'
                 }
             }
         ],
+        'requirePorts': True,
```

Comment:
Same as before, asking for a dynamic port + `requirePorts=true` shouldn't even be allowed by Marathon.